### DISCUSSION TITLE
manjaro moose conda conflicts

### DISCUSSION THREAD

**Discussion by zhanghg1983** [idaholab/moose#16650] [Q&A Getting Started] [answered]
Hi,guys,I am a new moose and conda user,
and my OS is manjaro ,
all I want to is to install moose in my OS ,successfully
but I almost mess up after many times try.
the error was like:
unsatisfiableError: The following specifications were found to be incompatible with each other:
Output in format: Requested package -> Available versions
Thanks for any help
[mooseerror.txt](https://github.com/idaholab/moose/files/5789831/mooseerror.txt)
[mooseerror1.txt](https://github.com/idaholab/moose/files/5789832/mooseerror1.txt)

**Comment by milljm** [comment, marked as answer]
I think you need to include the conda-forge channel (add an additional `--channel conda-forge` to your command). Hard to say though. Conflict resolution and displaying the conflict, is a rough go for Conda (in my honest opinion).

Also, I highly suggest you not employ the use of `sudo` when using `conda`. It should not be necessary, unless you are attempting to administer a multi-user machine, wishing to make the Conda packages you are about to install, available to everyone on the machine.

Very rarely have I seen this requirement. Most folks enjoy installing their own instance of Conda to their home directories. Making them the owner (without needing sudo).

**Reply by zhanghg1983** [reply]
Hi,milljm, Thank you very much for the help. So sorry for  replying so lately. after receving your suggestions. I tried another several times,but still failed. 
Today,I went through some blogs about conda, and someone said **delete the **~/.condarc**** may work.
I tried and it works, still don't know why. hopefully It will offer some help for the ones with simliar problems.
